### DISCUSSION TITLE
Error Scrapping Twitter

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#674] [Q&A] [answered]
Saya coba mereview materi DSS Twitter Sentiment Analysis. Namun terdapat error saat melakukan scrappingnya : 
<img width="800" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/c50294cd-6bb9-44cd-8b2c-0ea648ddaa28">

Mohon bantuannya kak, terima kasih 🙏

**Comment by jafarhf1** [comment, marked as answer]
Dear Pak @yohanespm77 

Untuk saat ini kebetulan package `stweet`  sudah tidak dapat digunakan untuk melakukan scrapping twitter. Terlihat juga dari errornya terdapat message: Forbidden yang artinya kemungkinan sudah diblok oleh pihak twitternya

Terima kasih.
